torch/package/_importlib.py

```diff
@@ -61,7 +61,7 @@ def _calc___package__(globals):
     spec = globals.get("__spec__")
     if package is not None:
         if spec is not None and package != spec.parent:
-            _warnings.warn(
+            _warnings.warn(  # noqa: G010
```

Comment:
lol, what's this?! Is the linter not smart enough

Reply:
Unfortunately, yeah. I don't know why we import the internals _warning module here instead of the normal one lol.

Reply:
@ezyang Do you know why this codepath imports `_warnings` instead of normal `warnings` https://github.com/python/cpython/blob/3.11/Lib/warnings.py not sure if this is intentional or not.

Reply:
this is a polyfill for importlib, looks like. So we may not be able to assume that the Python warnings.py actually exists. Best not to change it.